src/rest/DataQueryEndpoints.js

```diff
@@ -16,8 +16,10 @@ function onDataFetchDone(res, dataPoints, wrapper, content, volumeLogger) {
             let volumeBytes = 0
             res.send(dataPoints.map((streamMessage) => {
                 volumeBytes += streamMessage.getSerializedContent().length
+                if (streamMessage.version === 30) {
```

Comment:
Would be important to get rid of the version logic here, to keep it future-proof. Would be nice to use `streamMessage.serialize()` here, but I guess we don't want an array of strings as the result. 

What if we extend `streamMessage.serialize()` to take a new boolean param `stringify`? By default it would be true, but if it's false, it won't `JSON.stringify` the result object/array.

`serialize(version = VERSION, stringify = true) { ... }`